web/controllers/class_controller.ex

```diff
@@ -51,12 +51,12 @@ defmodule CoursePlanner.ClassController do
     end
   end
 
-#  def delete(conn, %{"id" => id}) do
-#    class = Repo.get!(Class, id)
-#    ClassHelper.delete(class)
-
-#    conn
-#    |> put_flash(:info, "Class deleted successfully.")
-#    |> redirect(to: class_path(conn, :index))
-#  end
+  def delete(conn, %{"id" => id}) do
+    class = Repo.get!(Class, id)
```

Comment:
You can put the get inside the `ClassHelper` module to remove the `Repo` dependency from the controller.